paperscraper/lib.py

```diff
@@ -121,27 +125,36 @@ async def link_to_pdf(url, path, session: ClientSession) -> None:
             return
         # try to find a pdf link
         html_text = await r.text()
-        # try for chemrxiv special tag
-        pdf_link = re.search(
-            r'content="(https://chemrxiv.org/engage/api-gateway/chemrxiv/assets.*\.pdf)"',
-            html_text,
-        )
-        if pdf_link is None:
+
+        # I know this looks weird
+        # I just need to try stuff and be able
+        # to break out of flow if I find a pdf
+        for _ in range(1):
```

Comment:
For something like this, a closure function can also be nice

Reply:
Probably a better idea - will do that. Thanks!